tests/manifests/vm_with_sidecar_hook.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: kubevirt.io/v1alpha2
 kind: VirtualMachineInstance
 metadata:
   annotations:
-    hooks.kubevirt.io/hookSidecars: '[{"image": "phoracek/example-hook-sidecar"}]'
+    hooks.kubevirt.io/hookSidecars: '[{"image": "kubevirt/example-hook-sidecar:v0.12.0-alpha.2"}]'
```

Comment:
@cynepco3hahue , suggested we update the sidecar here and raise an issue on kubevirt , for now normal error message for the failures that we see. Will raise an issue shortly for this.